### DISCUSSION TITLE
file "ModelConfiguration.ini" in work folder

### DISCUSSION THREAD

**Discussion by TLSY2** [flybywiresim/aircraft#4421] [Help] [answered]
Hi, I opened an issue and it got closed, the moderator i think solved it but I cant find anywhere the folder he told me to access. Does anyone know where it is and can help me?

I dont know where to find ModelConfiguration.ini and he says create/open, But i dont understand, I wrote a reply but he didnt respond yet and its been like 3 days, he used to respond very fast...

Here's the text from him:
I've added a configuration parameters that allows the configuration of the key events. The example shows default config for all flight controls. You can also skip the ones you don't wont to configure, they are then using defaults.

Create/open file "ModelConfiguration.ini" in work folder and add the following section:

[FLIGHT_CONTROLS]
KEY_CHANGE_AILERON = 0.02
KEY_CHANGE_ELEVATOR = 0.02
KEY_CHANGE_RUDDER = 0.02

**Comment by aguther** [comment, marked as answer]
In section "Reporting Issues" you find the location for the work folders depending on your version:
https://docs.flybywiresim.com/start/autopilot-fbw/

Beside that it turned out that there is currently an issue with the SDK that the aileron events are not working as they should.
